src/igvfd/schemas/changelogs/rodent_donor.md

```diff
@@ -4,6 +4,7 @@
 
 * *taxon_id* now restricted to NCBI taxonomy ids that start with NCBI:txid followed by numbers. Example NCBI:txid10090 for *Mus musculus*
 * *taxon_id* replaced by organism
-* defined *sex* as required property and defaulted it to *unspecified*
-* rodent_donor has aliases mixin and submitted mixin removed, but still has properties for aliases and submitted due to change of basic_item mixin.  The actual properties have not changed.
-* organism property renamed to taxa
+* Defined *sex* as required property and defaulted it to *unspecified*
+* *rodent_donor* has *aliases* mixin and *submitted* mixin removed, but still has properties for *aliases* and *submitted* due to change of basic_item mixin.  The actual properties have not changed.
```

Comment:
please don't modify changelog properties unrelated to the ticket

Reply:
I didn't modify technically. I just changed the text formatting since I wanted it to be uniform and how we did it on encoded.